src/main/java/com/shippo/ExampleOrder.java

```diff
@@ -0,0 +1,86 @@
+package com.shippo;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.time.LocalDateTime;
+
+import com.shippo.exception.ShippoException;
+import com.shippo.model.Order;
+
+public class ExampleOrder {
+
+	public static void main(String[] args) throws ShippoException {
+		
+		// replace with your Shippo Token
+		// don't have one? get more info here (https://goshippo.com/docs/#overview)
+		Shippo.setApiKey("shippo_test_cf1b6d0655e59fc6316880580765066038ef20d8");
+		Shippo.setApiVersion("2018-02-08");
+
+		// Optional defaults to false
+		//Shippo.setDEBUG(true);
+
+		// to address
+		Map<String, Object> toAddressMap = new HashMap<String, Object>();
+		toAddressMap.put("name", "Mr Hippo");
+		toAddressMap.put("company", "Regents Park");
+		toAddressMap.put("street1", "Outer Cir");
+		toAddressMap.put("city", "London");
+		toAddressMap.put("state", null);
+		toAddressMap.put("zip", "NW1 4RY");
+		toAddressMap.put("country", "GB");
+		toAddressMap.put("phone", "+1 555 341 9393");
+		toAddressMap.put("email", "ms-hippo@goshipppo.com");
+		toAddressMap.put("is_residential", false);
+		toAddressMap.put("metadata", "For Order Number 123");
+
+		// from address
+		Map<String, Object> fromAddressMap = new HashMap<String, Object>();
+		fromAddressMap.put("name", "Ms Hippo");
+		fromAddressMap.put("company", "San Diego Zoo");
+		fromAddressMap.put("street1", "2920 Zoo Drive");
+		fromAddressMap.put("city", "San Diego");
+		fromAddressMap.put("state", "CA");
+		fromAddressMap.put("zip", "92101");
+		fromAddressMap.put("country", "US");
+		fromAddressMap.put("email", "mshippo@goshipppo.com");
+		fromAddressMap.put("phone", "+1 619 231 1515");
+		fromAddressMap.put("metadata", "Customer ID 123456");
+
+		// parcel
+		Map<String, Object> lineItem = new HashMap<String, Object>();
+        lineItem.put("title", "Demo Line Item Object");
```

Comment:
some spacing issues probably from IDE weirdness

Reply:
Spaces & Tabs used in different files, I've installed a linter to fix this